pkg/services/rendering/rendering.go

```diff
@@ -22,7 +22,11 @@ import (
 
 func init() {
 	remotecache.Register(&RenderUser{})
-	registry.RegisterService(&RenderingService{})
+	registry.Register(&registry.Descriptor{
```

Comment:
This could potentially cause an issue with plugin loader, have you verified it works as expected when using renderer plugin?

Reply:
I'm using the rendering plugin sym linked into the plugins folder right now and it works nicely. But I can test with the released beta as well to make sure, not sure if that could work differently?

Reply:
Works with an installed plugin as well.